### ISSUE TITLE
Sentry captures sent via signal handler don't group

### ISSUE THREAD

**Issue by jdunck** [getsentry/raven-python#596]
We get some raven captures out of signal handlers, e.g. 

![screen shot 2015-04-09 at 11 13 15 am](https://cloud.githubusercontent.com/assets/10229/7073609/7b7f2f12-dea9-11e4-8f6d-f88ae655dd91.png)

This is likely caused by OOM, but because the app-level stack is different on each of the OOM occurrences, the captures don't group.  

I think it would be good to have a convenience method around capture-from-signal-handler which clues raven into disregarding the stack for grouping purposes.  

I'm not clueful enough about raven-python internals to easily implement this -- could you give me a pointer about what `.capture()` params I should manipulate in the convenience wrapper?

**Comment by jdunck** [comment]
(Possibly these are actually request-timeout-exceeded, but the grouping point stands.)

**Comment by xordoquy** [comment]
Grouping is done by the text sent to sentry. The exception titre wwould probably help here

**Comment by jdunck** [comment]
'gunicorn.workers.base in handle_abort'

But it clearly doesn't group *only* by the exception title...

**Comment by dcramer** [comment]
This makes sense. We actually were having convos about certain kinds of automated grouping behavior today. This one would be solved by us saying "group all exceptions like this together", but the solution I have in mind atm requires you to put the pattern into the server somewhere (i.e. in a textbox). Think that's reasonable enough here?

**Comment by jdunck** [comment]
Yeah, that would work OK - I was thinking exceptions from signal handlers were a different use case than typical sentry, and often harder to debug (OOM, timeout, HUP, etc.), so a smarter client might be justifiable.  But regex would also work in a pinch.

**Comment by dcramer** [comment]
Super slow here, but I agree that we can special case signal handlers. We might want to do this from an API POV, otherwise we have to deal with checksum on these specifics. In the upcoming API change we're going to add hints, but we could always add them in the current iteration and do a minor version change.
